spec.html

```diff
@@ -25935,20 +25935,23 @@
       </emu-clause>
 
       <!-- es6num="20.3.1.7" -->
-      <emu-clause id="sec-local-time-zone-adjustment">
-        <h1>Local Time Zone Adjustment</h1>
-        <p>An implementation of ECMAScript is expected to determine the local time zone adjustment. The local time zone adjustment is a value <dfn>LocalTZA</dfn> measured in milliseconds which when added to UTC represents the local <em>standard</em> time. Daylight saving time is <em>not</em> reflected by LocalTZA.</p>
+      <emu-clause id="sec-time-zone-offset-from-utc">
+        <h1>Time Zone Offset from UTC</h1>
+        <p>An implementation of ECMAScript is expected to determine the local time zone offset from UTC TZOffsetUTC(_t_) using the best available information on time zones. The time zone offset from UTC at time represented by time value _t_ (UTC) is a value <dfn>TZOffsetUTC(_t_)</dfn> measured in milliseconds which when subtracted from UTC represents the local time.</p>
         <emu-note>
           <p>It is recommended that implementations use the time zone information of the IANA Time Zone Database <a href="http://www.iana.org/time-zones/">http://www.iana.org/time-zones/</a>.</p>
         </emu-note>
       </emu-clause>
 
       <!-- es6num="20.3.1.8" -->
-      <emu-clause id="sec-daylight-saving-time-adjustment">
-        <h1>Daylight Saving Time Adjustment</h1>
-        <p>An implementation dependent algorithm using best available information on time zones to determine the local daylight saving time adjustment DaylightSavingTA(_t_), measured in milliseconds. An implementation of ECMAScript is expected to make its best effort to determine the local daylight saving time adjustment.</p>
+      <emu-clause id="sec-time-zone-offset-from-localtime">
+        <h1>Time Zone Offset from LocalTime</h1>
+        <p>An implementation of ECMAScript is expected to determine the local time zone offset from local time TZOffsetLocal(<emu-eqn>_t_<sub>local</sub></emu-eqn>) using the best available information on time zones. The time zone offset at the local time <emu-eqn>_t_<sub>local</sub></emu-eqn> is a value <dfn>TZOffsetLocal(<emu-eqn>_t_<sub>local</sub></emu-eqn>)</dfn> measured in milliseconds which when added to the local time represents UTC.</p>
+        <p>When <emu-eqn>_t_<sub>local</sub></emu-eqn> represents wall time repeating multiple times at a positive time zone offset transition (e.g. when the daylight savings time ends or the time zone offset is increased) or skipped wall time at a negative time zone offset transitions (e.g. when the daylight savings time starts or the time zone offset is decreased), <emu-eqn>_t_<sub>local</sub></emu-eqn> should be interpreted with the time zone offset after the transition.</p>
```

Comment:
No need for "multiple times" - given that "repeating" implicitly says that - and explicitly saying "multiple times" rather than "twice" suggests more than twice, which does never happen and we have no sane way to cope with it if it did.

also: s/the daylight savings time/daylight-saving time/g
... and, at the end, s/after/before/, but description of the argument "prior" should cover this.